addons/crm/models/crm_lead.py

```diff
@@ -2542,16 +2542,18 @@ def _pls_get_safe_fields(self):
 
     # Compute Automated Probability Tools
     # -----------------------------------
-    def _pls_get_won_lost_total_count(self, team_results):
+    def _pls_get_won_lost_total_count(self, team_results, team_id=False):
         """ Get all won and all lost + total :
                first stage can be used to know how many lost and won there is
                as won count are equals for all stage
                and first stage is always incremented in lost_count
         :param frequencies: lead_scoring_frequencies
```

Comment:
Maybe add a small explanation for this new parameter (the rest of this docstring will be updated in master soon)

Reply:
Note that this is not stable compliant: if someone has an override, their module is broken.

Reply:
Ah yes, good catch. Maybe for stable, we could add team_id in the team_results before calling the method, use that value, and remove it once the call is done? (this method will probably disappear in master at some point, work in progress)

Reply:
Or, to avoid changing the content of the dict, we could instead do no modification in this function, but when the method returns 0,0,0 we do another check with team_id in the loop of _pls_get_naive_bayes_probabilities. This would probably be a totally stable-compliant solution :)